<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Factory</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="boxProject">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>182</width>
         <height>197</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Project</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>58</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>New</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>58</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnOpenProject">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Recent</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbxMRUProjects">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>60</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btnNewProject">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>0</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Properties</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnPropertiesProject">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>90</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnArchiveProject">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>120</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>120</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Archive</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>150</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnCloseProject">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>150</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="boxEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>182</width>
         <height>197</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Edit</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTreeView" name="tvwProject"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="boxBuild">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>182</width>
         <height>197</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Build</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="boxRun">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>182</width>
         <height>197</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Run</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="boxSettings">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>182</width>
         <height>197</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Settings</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnEdit">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Factory.qrc">
          <normaloff>:/icons/icons/Pen.png</normaloff>:/icons/icons/Pen.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTerm">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Factory.qrc">
          <normaloff>:/icons/icons/Computer.png</normaloff>:/icons/icons/Computer.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBrowser">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Factory.qrc">
          <normaloff>:/icons/icons/Globe.png</normaloff>:/icons/icons/Globe.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBuild">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Factory.qrc">
          <normaloff>:/icons/icons/Under Construction.png</normaloff>:/icons/icons/Under Construction.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSettings">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Factory.qrc">
          <normaloff>:/icons/icons/Gear.png</normaloff>:/icons/icons/Gear.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Factory.qrc"/>
 </resources>
 <connections/>
</ui>
